### DISCUSSION TITLE
Phone number display name is not working

### DISCUSSION THREAD

**Discussion by venkatesh1146** [manojadams/mui-forms#84] [Q&A] [answered]
There is a repro in this project : https://github.com/venkatesh1146/mui-forms-test-project

**Comment by manojadams** [comment, marked as answer]
@venkatesh1146 @venkateshpullaganti This has been fixed in the following version:
[mui-forms@2.0.7](https://www.npmjs.com/package/mui-forms/v/2.0.7)

**Reply by venkatesh1146** [reply]
Will this have all the fixes so far? 

Currently we are using this version : `mui-forms-2.0.13.tgz`
